packages/template-retail-react-app/app/static/dwac-21.7.js

```diff
@@ -0,0 +1,475 @@
+// Enhance the dw.ac namespace
```

Comment:
Should this script be part of the PWA_Kit project? Could we not download it from the same place that SFRA downloads it from? I'm asking this because I'm assuming that this file is now a copy of `dwac-xx.x.js` and if there was a bug in that script, you'd now have to fix it in multiple places.